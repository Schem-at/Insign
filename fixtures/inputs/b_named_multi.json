[
  { "pos": [0, 64, 0], "text": "@cpu.core=rc([100,70,-20],[104,72,-18])\n#cpu.core:logic.clock_hz=4" },
  { "pos": [10, 64, 10], "text": "@cpu.core=rc([100,73,-20],[104,75,-18])\n#cpu.core:logic.cache_mb=2" }
]
