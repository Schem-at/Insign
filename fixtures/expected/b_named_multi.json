{
  "cpu.core": {
    "bounding_boxes": [
      [
        [
          100,
          134,
          -20
        ],
        [
          104,
          136,
          -18
        ]
      ],
      [
        [
          110,
          137,
          -10
        ],
        [
          114,
          139,
          -8
        ]
      ]
    ],
    "metadata": {
      "logic.cache_mb": 2,
      "logic.clock_hz": 4
    }
  }
}
